### PR TITLE
docs(changelog): scalingo-18 v16 release

### PR DESCRIPTION
**Pull request by EtienneM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,13 +70,6 @@ object(s)`. Then, insert it with:
 {% include mdl_img.html %}
 ```
 
-Update the list of packages of the Docker image `scalingo-18`:
```
Comment (brandon-welsch) [inline README.md:73]: Why did you remove it, it's no longer relevant?

**Review by brandon-welsch — APPROVED** [review]
A comment otherwise LGTM

File: README.md (hunk 1 above)
Comment (EtienneM) [reply README.md:73]: As stated in my commit message, it has been moved to Notion, in the procedure to release a new stack version. I don't think it's really relevant here
